browser-test/src/application_review.test.ts

```diff
@@ -44,21 +44,27 @@ describe('normal application flow', () => {
 
     const applicantQuestions = new ApplicantQuestions(page);
 
+    // Applicant fills out first application block.

```

Comment:
praise: thank you for adding these comments! it really helps me figure out at a glance what's going on in the test